### PR TITLE
BERTopic hub integration announcement post

### PR DESCRIPTION
**Pull request by davanstrien**
TODO:
- [x] update blog post images

### CODE REVIEW

**Comment by davanstrien** [comment]
cc @osanseviero @yjernite I've aimed to include suggestions from both of you for this post. I'm not able to merge https://huggingface.co/datasets/huggingface/documentation-images/discussions/104 myself but will update URLs for images once that is merged.

**Review by osanseviero — COMMENTED** [review]
This looks neat!

File: bertopic.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+---
+
+<h1> Introducing BERTopic Integration with Hugging Face Hub"</h1> 
+
+<!-- {blog_metadata} -->
+<!-- {authors} -->
+
+We are thrilled to announce a significant update to the [BERTopic](https://maartengr.github.io/BERTopic) Python library, expanding its capabilities and further streamlining the workflow for topic modelling enthusiasts and practitioners. BERTopic now supports pushing and pulling trained topic models directly to and from the Hugging Face Hub. This new integration opens up exciting possibilities for leveraging the power of BERTopic in production use cases with ease.
+
+## What is BERTopic?
```
[the remark this answers is not in the document]
Comment (davanstrien) [reply bertopic.md:21]: Have added a short para at the start on this

**Comment by davanstrien** [comment]
@osanseviero I have implemented your suggestions now. Are you happy for me to merge this?

**Review by julien-c — COMMENTED** [review]
cool blogpost, thanks @MaartenGr @davanstrien!

**Pull request merged**